packages/crashlytics/android/src/main/java/io/invertase/firebase/crashlytics/ReactNativeFirebaseCrashlyticsModule.java

```diff
@@ -88,23 +92,7 @@ public void setAttributes(ReadableMap keyValuesMap, Promise promise) {
   @ReactMethod
   public void setUserId(String userId, Promise promise) {
     if (ReactNativeFirebaseCrashlyticsInitProvider.isCrashlyticsCollectionEnabled()) {
-      Crashlytics.getInstance().core.setUserIdentifier(userId);
-    }
-    promise.resolve(null);
-  }
-
-  @ReactMethod
-  public void setUserName(String userName, Promise promise) {
-    if (ReactNativeFirebaseCrashlyticsInitProvider.isCrashlyticsCollectionEnabled()) {
-      Crashlytics.getInstance().core.setUserName(userName);
-    }
-    promise.resolve(null);
-  }
-
-  @ReactMethod
-  public void setUserEmail(String userEmail, Promise promise) {
-    if (ReactNativeFirebaseCrashlyticsInitProvider.isCrashlyticsCollectionEnabled()) {
-      Crashlytics.getInstance().core.setUserEmail(userEmail);
+      FirebaseCrashlytics.getInstance().setUserId(userId);
```

Comment:
Similar to the init changes, are setUserName and setUserEmail going away? 
If functionality is being reduced or changed there should be some migration documentation

Reply:
https://firebase.google.com/docs/crashlytics/upgrade-sdk?platform=ios#setuseridentifier_is_now_setuserid_setusername_and_setuseremail_are_removed

> Reason for change
> We adopted the method name setUserID to be consistent with other Firebase APIs and removed setUserName and setUserEmail to **discourage logging PII through Crashlytics**.